File: I1_MVPAndRetrofit/app/src/main/java/com/example/tsvetelinastoyanova/weatherreportrevisited/weatherdetails/WeatherDetailsPresenter.java
package com.example.tsvetelinastoyanova.weatherreportrevisited.weatherdetails;

import android.util.Log;

public class WeatherDetailsPresenter implements WeatherDetailsContract.Presenter{
    @Override
    public void start() {
        Log.d("tag", "WeatherDetailsPresenter started");
    }
}

File: I1_MVPAndRetrofit_Kotlin/app/src/main/java/com/example/tsvetelinastoyanova/weatherapp/weatherdetails/weather/WeatherDetailsPresenter.kt
package com.example.tsvetelinastoyanova.weatherapp.weatherdetails.weather

import android.util.Log
import com.example.tsvetelinastoyanova.weatherapp.weatherdetails.weather.WeatherDetailsContract

class WeatherDetailsPresenter : WeatherDetailsContract.Presenter {
    override fun start() {
        Log.d("tag", "WeatherDetailsPresenter started")
    }
}